### ISSUE TITLE
Btrfs, target path already exists

### ISSUE THREAD

**Issue by sh3bang** [archlinux/archinstall#2649]
### Which ISO version are you using?

2.8.1

### The installation log

```
Hardware model detected: Dell Inc. XPS 13 9310; UEFI mode: True
Processor model detected: 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz
Memory statistics: 30949004 available out of 32574052 total installed
Could not detect virtual system: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [1]: none

System is not running in a VM: ['/usr/bin/systemd-detect-virt'] exited with abnormal exit code [1]: none

Virtualization detected: None; is VM: False
Graphics devices detected: dict_keys(['Intel Corporation TigerLake-LP GT2 [Iris Xe Graphics] (rev 01)'])
```

### describe the problem

Hello,

i am new to Arch-Linux (using Manjaro usually) and have a lot of trouble with archinstall and need some Help please, because python exceptions occur.

Case 1:
on my physical PC the setup suddenly interrupted, i dont now why exactly they want to touch nvme0n1p10:

`target path already exists`
[archinstall.log](https://github.com/user-attachments/files/16778952/archinstall.log)

Case 2:
even in my VM the setup dont complete:

`Unable to determine new partition number: {path}`
[archinstall-vm.log](https://github.com/user-attachments/files/16778950/archinstall-vm.log)

**Comment by Torxed** [comment]
Does this happen with archinstall [v2.8.5](https://github.com/archlinux/archinstall/releases/tag/v2.8.5) as well? I take it you're using v2.8.1.

**Comment by sh3bang** [comment]
yes. id did already a "pacman -Sy archinstall" (update to 2.8.5) and the error still occur.

**Comment by Torxed** [comment]
Haven't read thoroughly yet, but initial hint:

it looks like you're re-using partitions.
Because the disk was never wiped - and it's trying to re-create subvolumes specified during manual partitioning.

**Comment by sh3bang** [comment]
yes, i have also other partitions and Linux distributions installed and want to install Arch only on a specific partitions.

But even on Case 2 (see log file from above), where i installed a fresh VM with "guided partitions" (don't know the correct term) the setup failed

**Comment by Torxed** [comment]
We should handle those scenarios, so we'll have to debug and fix this obviously.
It was just preliminary debugging information to find the root cause.
The second one is a bit surprising tho, that should work.

**Comment by sh3bang** [comment]
it`s easy to reproduce! Download Arch-Linux ISO, create a new KVM, run archinstall, choose "Suggest partition layout" and maybe ext4 as FS and Setup will fail.
